crates/http-sink/src/main.rs

```diff
@@ -6,15 +6,16 @@ use config::HttpConfig;
 use futures::SinkExt;
 use sink::HttpSink;
 
-use fluvio_connector_common::{connector, consumer::ConsumerStream, tracing::debug, Sink};
+use fluvio_connector_common::{connector, consumer::ConsumerStream, tracing, Sink};
 
 #[connector(sink)]
 async fn start(config: HttpConfig, mut stream: impl ConsumerStream) -> Result<()> {
-    debug!(?config);
+    tracing::debug!(?config);
 
     let sink = HttpSink::new(&config)?;
     let mut sink = sink.connect(None).await?;
 
+    tracing::info!("Starting HTTP Sink Connector");
     while let Some(item) = stream.next().await {
```

Comment:
maybe we should add a `debug!` here. With: received record

Reply:
Oh! Bors just merged! Let me open another!